<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SliderBar settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QListWidget" name="listCategories">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>Serial port</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Plugins</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="settingsCommunications">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="titleCommunications">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Communications</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTabWidget" name="tabWidgeCommunications">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tabUSBDevice">
          <attribute name="title">
           <string>USB Device</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0" colspan="2">
            <widget class="QGroupBox" name="additionalOptionsGroupBox">
             <property name="title">
              <string>Additional options</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <widget class="QCheckBox" name="eAutoconnect">
                <property name="text">
                 <string>Auto connect on startup</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabProtocol">
          <attribute name="title">
           <string>Protocol</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QGroupBox" name="grpProtocolFlags">
             <property name="title">
              <string>Protocol flags</string>
             </property>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="lStartFlag">
                <property name="text">
                 <string>Start flag</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="eStartFlag">
                <property name="prefix">
                 <string>0x</string>
                </property>
                <property name="maximum">
                 <number>255</number>
                </property>
                <property name="value">
                 <number>18</number>
                </property>
                <property name="displayIntegerBase">
                 <number>16</number>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="lEndFlag">
                <property name="text">
                 <string>End flag</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="eEndFlag">
                <property name="prefix">
                 <string>0x</string>
                </property>
                <property name="maximum">
                 <number>255</number>
                </property>
                <property name="value">
                 <number>19</number>
                </property>
                <property name="displayIntegerBase">
                 <number>16</number>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="lEscapeFlag">
                <property name="text">
                 <string>Escape flag</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QSpinBox" name="eEscapeFlag">
                <property name="prefix">
                 <string>0x</string>
                </property>
                <property name="maximum">
                 <number>255</number>
                </property>
                <property name="value">
                 <number>125</number>
                </property>
                <property name="displayIntegerBase">
                 <number>16</number>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="lXORFlag">
                <property name="text">
                 <string>XOR flag</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QSpinBox" name="eXORFlag">
                <property name="prefix">
                 <string>0x</string>
                </property>
                <property name="maximum">
                 <number>255</number>
                </property>
                <property name="value">
                 <number>32</number>
                </property>
                <property name="displayIntegerBase">
                 <number>16</number>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="lACKFlag">
                <property name="text">
                 <string>ACK flag</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QSpinBox" name="eACKFlag">
                <property name="prefix">
                 <string>0x</string>
                </property>
                <property name="maximum">
                 <number>255</number>
                </property>
                <property name="value">
                 <number>251</number>
                </property>
                <property name="displayIntegerBase">
                 <number>16</number>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="lNACKFlag">
                <property name="text">
                 <string>NACK Flag</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QSpinBox" name="eNACKFlag">
                <property name="prefix">
                 <string>0x</string>
                </property>
                <property name="maximum">
                 <number>255</number>
                </property>
                <property name="value">
                 <number>252</number>
                </property>
                <property name="displayIntegerBase">
                 <number>16</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <widget class="QGroupBox" name="grpAck">
             <property name="title">
              <string>Acknowledgement</string>
             </property>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="lAckTimeOut">
                <property name="text">
                 <string>Ack timeout</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="eAckTimeOut">
                <property name="suffix">
                 <string> ms</string>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>1000</number>
                </property>
                <property name="value">
                 <number>15</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="settingsPlugins">
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="titlePlugins">
         <property name="text">
          <string>Plugin settings</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTabWidget" name="tabWidgetPlugins">
         <widget class="QWidget" name="Plugin1">
          <attribute name="title">
           <string>Plugin 1</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Reset|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
